src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greet = showWelcomeScreen();
+        System.out.println(greet);
+        
+        Task currentTask;
+        Task[] storeTasks = new Task[100];
```

Comment:
Should the size 100 be extracted as a constant?

Reply:
Thank you! will edit it later